crates/ruff_linter/resources/test/fixtures/flake8_quotes/doubles_escaped_unnecessary.py

```diff
@@ -0,0 +1,45 @@
+this_should_raise_Q100 = 'This is a \"string\"'
```

Comment:
@charliermarsh you may want to rename the `Q100` references to `Q004` in these two files as well ;)

Reply:
Thank you!